### DISCUSSION TITLE
My Application is not getting processed

### DISCUSSION THREAD

**Discussion by sohamdeogaonkar** [community/community#54836] [GitHub Education]
### Hi everyone,

![image](https://user-images.githubusercontent.com/63729589/236753314-d58877d4-3fd1-4f42-93f6-3c95177d76be.png)

I am not getting whether they checking my application or not.

I am getting this error with in 4-5 second after pressing process button.

Why my application is not getting processed manually.

I have tried to upload as clear as possible image i can.
But i am unable to focus properly to document
and there is only one front camera option and also dont have options to upload documents

**Comment by coolacpc** [comment]
Same problem here.
Even opened a new ticket with support, but no answer so far 😔

**Reply by devashish2024** [reply]
1. Having great quality is cool, however prefer a low but readable quality (for a human, computer will understand it since your ID card was printed with printer and not written)
2. Wait some time
3. Do not take front photo exactly - this appears to be fake because several sites like Canva show you a image of a fake 3D model of your design, which can make them reject it, so be a bit different faced
4. Background: white and solid backgrounds are sussy, do not even use background remover, the brightness should be good. If you put it on your bed I got more chances to quick (instantly) approved damn

**Comment by iamreiyn** [comment]
Hi @sohamdeogaonkar @coolacpc 

Frequently, you may see this typical warning during pre-processing as GitHub detects inconsistencies between your profile details and the proof you have provided. The verification procedure examines several factors, including your GitHub username, email, and name, to verify the submitted evidence, so it's essential to ensure that they match to proceed.

If you were unable to upload the documents, try doing it from your computer browser or enable "Desktop mode" in your phone's browser and check.

**Reply by sohamdeogaonkar** [reply]
We dont have upload option on destop mode.

I also tried through adding OBS virtual cam but it is not showing the option to switch to obs

**Reply by sohamdeogaonkar** [reply]
> Hi @sohamdeogaonkar @coolacpc
> 
> Frequently, you may see this typical warning during pre-processing as GitHub detects inconsistencies between your profile details and the proof you have provided. The verification procedure examines several factors, including your GitHub username, email, and name, to verify the submitted evidence, so it's essential to ensure that they match to proceed.
> 
> If you were unable to upload the documents, try doing it from your computer browser or enable "Desktop mode" in your phone's browser and check.

Oh I FInally get way to upload by drag and drop

but
![image](https://user-images.githubusercontent.com/63729589/237007148-328b0925-60fa-4ac6-b366-c537a20f7bca.png)

**Comment by mishra-prajjawal** [comment]
Hi @sohamdeogaonkar 👋, 
The problem you are facing can be frustrating, So thank you keeping your patience and reaching out for suggestions. 
The problem you are facing is linked to one or some certain points given below :
1. Check if your institute exists in the [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education).<br> **If your school or institute is not in the program** then you are likely to be rejected.<br> 
3. Check your details passed when applying,**Improper or Invalid** details can also be a root cause to be rejected. <br>for example: <br> Your **student ID** card may not be up to date or the details might not match etc.<br>
4. **Using a VPN**, Many students use VPN's during signup which can cause issues to GitHub. Hence it is required to have your VPN turned off. <br> 
5. **Blurry or Unclear Images or documents** sent also come in handy to reject your application, Hence it is advised to use a good quality picture to upload or a good quality camera to take picture based on what is asked during signup . <br>
6. **Not Eligible**, If you are a student but not at least 13 years old then you are rejected until you are eligible.
7. **Illegal Activities** trying malicious ways to be approved can also be considered both as a criminal activity and violation to GitHub Terms Of Service, Then GitHub can actively decline services and even terminate the account.
--- 
I am tagging some pages which you may browse, They may be helpful:
1. [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)
2. [My Student Developer Pack application was rejected #51322](https://github.com/orgs/community/discussions/51322)
3. [Getting rejected every time I applied for github student developer pack (renew) #17808](https://github.com/orgs/community/discussions/17808)
4.  [Failed Github Student Developer Pack! #36912](https://github.com/orgs/community/discussions/36912)
5. [Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)
6. [Apply to GitHub Global Campus as a student - GitHub Docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)
7. [How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/#ive-been-rejected-but-im-really-a-student-what-should-i-do)

Thanks for reaching out again

**Reply by sohamdeogaonkar** [reply]
If my school is not registered in the programme.

So
Does I am not unable to use student benifits?

I thought it is another way where school can also apply.

But
Is it Compulsory to have my school registered

**Reply by sohamdeogaonkar** [reply]
> Hi @sohamdeogaonkar 👋, The problem you are facing can be frustrating, So thank you keeping your patience and reaching out for suggestions. The problem you are facing is linked to one or some certain points given below :
> 
> 1. Check if your institute exists in the [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). **If your school or institute is not in the program** then you are likely to be rejected.
> 2. Check your details passed when applying,**Improper or Invalid** details can also be a root cause to be rejected. for example:  Your **student ID** card may not be up to date or the details might not match etc.
> 3. **Using a VPN**, Many students use VPN's during signup which can cause issues to GitHub. Hence it is required to have your VPN turned off.
> 4. **Blurry or Unclear Images or documents** sent also come in handy to reject your application, Hence it is advised to use a good quality picture to upload or a good quality camera to take picture based on what is asked during signup .
> 5. **Not Eligible**, If you are a student but not at least 13 years old then you are rejected until you are eligible.
> 6. **Illegal Activities** trying malicious ways to be approved can also be considered both as a criminal activity and violation to GitHub Terms Of Service, Then GitHub can actively decline services and even terminate the account.
> 
> I am tagging some pages which you may browse, They may be helpful:
> 
> 1. [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)
> 2. [My Student Developer Pack application was rejected #51322](https://github.com/orgs/community/discussions/51322)
> 3. [Getting rejected every time I applied for github student developer pack (renew) #17808](https://github.com/orgs/community/discussions/17808)
> 4. [Failed Github Student Developer Pack! #36912](https://github.com/orgs/community/discussions/36912)
> 5. [Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)
> 6. [Apply to GitHub Global Campus as a student - GitHub Docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)
> 7. [How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/#ive-been-rejected-but-im-really-a-student-what-should-i-do)
> 
> Thanks for reaching out again



Ok
Now I managed to improve my image quality 

But
Still
It is getting rejected within seconds


Why it is not going to be verified manually 

Anf if there is any issue
Why not saying clearly whats the issue.
Just giving the link of all possible reason

Why cant tell the particular reason why it is automatically getting rejected

**Reply by mishra-prajjawal** [reply]
Hi @sohamdeogaonkar, 
>If my school is not registered in the programme.
>So
>Does I am not unable to use student benifits?
>I thought it is another way where school can also apply.
>But
>Is it Compulsory to have my school registered

It is not mandatory for you to be in a school that is in enlisted The GitHub Campus but if your school is enlisted then it gives you a high chance to be approved.
 
>Ok
Now I managed to improve my image quality
>But
Still
It is getting rejected within seconds
Why it is not going to be verified manually
Anf if there is any issue
Why not saying clearly whats the issue.
Just giving the link of all possible reason
Why cant tell the particular reason why it is automatically getting rejected

Sorry to hear that your account could not be approved. 
You see there are 100 Million+ GitHub Active users and there are Millions of Students around the world enjoying and learning [The GitHub Student Developer Pack](https://education.github.com/pack), So it is going to expensive in terms of money, time and manpower. Hence whole the process is Automated. 
Coming to your next question, 
The General reasons provided after the rejection is because the system is not designed to produce output based on what the error is and why is it, It is designed to check and validate the details passed and the documents sent, If the details looks good it gives a green light else it will signal red and reject with common issues people face while applying.
So sorry again we cannot help you there. 

Thank you for your patience.

**Comment by sohamdeogaonkar** [comment]
![image](https://user-images.githubusercontent.com/63729589/237016709-9fe522e2-55fd-4b12-aeb6-b87d3a0f4697.png)

now
i have this error 

but
i delected that account because it was get suspened due to use of multiple accounts

so to keep one account only.
i delected that account

**Reply by mishra-prajjawal** [reply]
Hi @sohamdeogaonkar,
In order for GitHub to be safe and away from fraudulent access to it's GitHub Student Developer Pack your application documents are saved for future to allow only one unique document, Here the document is same as the one you gave earlier in the different account which got suspended due to not complying to [GitHub Terms of Service](https://docs.github.com/site-policy/github-terms/github-terms-of-service). 

For further enquiry you may contact [GitHub Support](mailto:support@github.com)

Thank you for your patience.

**Reply by sohamdeogaonkar** [reply]
> Hi @sohamdeogaonkar, In order for GitHub to be safe and away from fraudulent access to it's GitHub Student Developer Pack your application documents are saved for future to allow only one unique document, Here the document is same as the one you gave earlier in the different account which got suspended due to not complying to [GitHub Terms of Service](https://docs.github.com/site-policy/github-terms/github-terms-of-service).
> 
> For further enquiry you may contact [GitHub Support](mailto:support@github.com)
> 
> Thank you for your patience.

what process should i follow now.

is it possible to have the access to plan
or i cant get it for whole life

**Reply by mishra-prajjawal** [reply]
> > Hi @sohamdeogaonkar, In order for GitHub to be safe and away from fraudulent access to it's GitHub Student Developer Pack your application documents are saved for future to allow only one unique document, Here the document is same as the one you gave earlier in the different account which got suspended due to not complying to [GitHub Terms of Service](https://docs.github.com/site-policy/github-terms/github-terms-of-service).
> > For further enquiry you may contact [GitHub Support](mailto:support@github.com)
> > Thank you for your patience.
> 
> what process should i follow now.
> 
> is it possible to have the access to plan or i cant get it for whole life

Hello @sohamdeogaonkar 
As far as I understand you cannot gain access to your account suspended and yes you cannot access the GitHub Student Developer pack until you are not approved. 
Lastly, No GitHub Student Developer pack is not for lifetime it is updated on  yearly basis.
Thanks

**Comment by samus-aran** [comment]
Hi @sohamdeogaonkar 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!

**Comment by Wolfyxon** [comment]
I'm not sure if my application was submitted. I haven't received any confirmation emails and going to the benefits page always redirects me to the *Upload proof of your academic status* even though I already took a photo and submitted the form.
I've already tried twice, is this normal?

**Reply by mpalasz12** [reply]
I have the same problem and don't know how to fix it.

**Reply by mpalasz12** [reply]
I just did a photo using my laptop camera and it worked for me, maybe it will work also for you.

**Reply by Wolfyxon** [reply]
That's what I did.

**Reply by ghost** [reply]
do you have the right resolution? I had the same problem, but resolved it by retaking the photo. Im not sure if that was the problem , just guessing.

**Reply by Wolfyxon** [reply]
What would be the *right* resolution?

**Reply by mpalasz12** [reply]
How long do you have to wait to gain access to copilot? I got approved on monday and still dont’t have it.
